js-OOP/classes1.js

```diff
@@ -0,0 +1,53 @@
+// Task 4
+class CarClass {
+    constructor(make, model, year) {
+      this.make = make;
+      this.model = model;
+      this.year = year;
+      this.speed = 0;
```

Comment:
speed property should be private 
```suggestion
      this.#speed = 0;
```